site/gatsby-site/src/api/lookupbyurl.js

```diff
@@ -0,0 +1,109 @@
+import Rollbar from 'rollbar';
+import siteConfig from '../../config';
+
+import Cors from 'cors';
+
+const cors = Cors();
+
+const isValidURL = (string) => {
+  try {
+    new URL(string);
```

Comment:
For a follow-up PR – I think we should consider matching on URLs regardless of whether they include protocol (only strings with `http(s)` are "valid" URLs) and regardless of including `www`. Yes, `www` is technically a sub-domain, but I think the user would expect that URLs are found all the same.

```js
a = new URL("https://nytimes.com") 
b = new URL("https://www.nytimes.com") 
console.log(a.host, b.host)
```
```
> nytimes.com www.nytimes.com
```